src/test/java/com/socrata/api/SodaImporterTest.java

```diff
@@ -499,13 +494,6 @@ public void testCreatingExternalDataset() throws IOException, SodaError, Interru
         final ExternalDataset fileDataset = (ExternalDataset) importer.createDataset(dataset1);
 
         try {
-            TestCase.assertNotNull(fileDataset);
```

Comment:
Changed a test assertion here--this expected an unpublished dataset after `importer.createDataset` is called, but it's actually already published. I removed extra assertions.